File: 0832-flipping-an-image/0832-flipping-an-image.rb
# @param {Integer[][]} image
# @return {Integer[][]}
def flip_and_invert_image(image)
    image.map(&:reverse).map do |ea|
        ea.join.gsub('0', '2').gsub('1', '0').gsub('2', '1').chars.map(&:to_i)
    end
end

File: 0832-flipping-an-image/0832-flipping-an-image.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        reversed_image =[row[::-1] for row in image]
        result = [[1 if el == 0 else 0 for el in row] for row in reversed_image]   
        return result
        